packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -105,7 +105,11 @@ export const defaultMyChatMessageContainer: ComponentSlotStyle = {
 export const defaultChatMessageContainer: ComponentSlotStyle = {
   maxWidth: '100%',
   minWidth: `${CHAT_MESSAGE_CONTAINER_MIN_WIDTH_REM}rem`,
-  marginRight: '0rem'
+  marginRight: '0rem',
+  ' img': {
```

Comment:
What does the whitespace in `' img'` do? (is it intentional?)

Reply:
Yes! This is to generate selector like "defaultContainer img" for limiting all messages inside of the chat bubble